backend/danswer/connectors/slack/connector.py

```diff
@@ -205,12 +205,17 @@ def thread_to_doc(
     "group_leave",
     "group_archive",
     "group_unarchive",
```

Comment:
this is necessary to make the tests work. i also dont think this will change the quality of the results but im not sure.